addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3399,7 +3401,9 @@ var SnippetsMenu = Widget.extend({
                             if (dropped) {
                                 $toInsert.detach();
                                 $toInsert.addClass('oe_snippet_body');
-                                $('.oe_drop_zone').removeClass('invisible');
+                                for (const dropzoneEl of self.$body[0].querySelectorAll(".oe_drop_zone")) {
+                                    dropzoneEl.classList.remove("invisible");
+                                }
```

Comment:
In this case, it seems to me that there is not the bug. Indeed, the drag and drop starts in the iframe.
I hesitate to make the change so as not to have to do a new search in the DOM also using the `$dropZones` variable but as it is a stable version it is perhaps not worth changing the line if it is not necessary (no bug).

Reply:
> In this case, it seems to me that there is not the bug

@bvr-odoo I think that the bug is here as well (see video). It can be explained because, the `SnippetsMenu` is outside of the iframe so it does not have access to the dropzones located inside of the iframe. However, to solve the problem, we could use the already defined `$dropZones` as you suggested it [here](https://github.com/odoo/odoo/pull/136644/commits/8c303a689d3d0ac59cf7d4e70ca6bd7b064ca70a#r1341093264)  :+1: 

[Screencast from 09-29-2023 11:18:55 AM.webm](https://github.com/odoo/odoo/assets/114914971/fb3f3bc0-bc94-4b88-95c8-c67c5b7b4648)

Reply:
Sorry I confused the two places where we remove the "invisible" class. What I meant is that the bug does not exist in the part "Called when the snippet is starting to be dragged thanks to the 'move' button."

Reply:
So doing
```js
$dropZones.forEach(dropzoneEl => dropzoneEl.classList.remove("invisible"));
```
could be enough I think

Reply:
> What I meant is that the bug does not exist in the part "Called when the snippet is starting to be dragged thanks to the 'move' button."

@bvr-odoo , Except if I misunderstood  something, I think that the bug also happens in this case ;) 

[Screencast from 09-29-2023 01:40:41 PM.webm](https://github.com/odoo/odoo/assets/114914971/4932a890-bcca-4be4-88eb-9d07530d0133)

Reply:
Indeed, I was misled because the line seems no longer necessary.. If I delete it, I have no bug ...
To investigate ...